src/i18n/fr_FR.json

```diff
@@ -0,0 +1,549 @@
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_ROOT_NOT_DEPLOYED_2" : "Vous pouvez aussi déployer et démarrer cette instance avec tous ses enfants en un clique.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_CYCLE" : "Le cycle de vie de cette instance est gérée par l'installeur  {{installer}} de Roboconf.",
+	
+	"INSTANCES_INCLUDES_ADDITIONAL_INFORMATION" : "Information additionnelle",
+	
+	"INSTANCES_INCLUDES_ACCESS_EXPORTED_VARIABLES" : "Accéder aux variables exportées",
+	
+	"INSTANCES_INCLUDES_STOP_INSTANCE" : "Arrêter cette instance",
+	
+	"INSTANCES_INCLUDES_START_INSTANCE" : "Start this instance",
+	
+	"INSTANCES_INCLUDES_UNDEPLOYED_IT" : "Annuler le déploiement de cette instance",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_DEPLOYED_STARTED" : "L'instance a démarré. Elle ne peut qu'être stoppée.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_DEPLOYED_STOPPED" : "L'instance est arrêtée. Elle peut soit être redémarrée ou en mode sans déploiement",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_DEPLOYING" : "L'instance va être déployée par l'installeur {{installer}} de Roboconf.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_DEPLOYING_2" : "Cela peut prendre quelques temps.",
+	
+	"INSTANCES_INCLUDES_PARENT_INSTANCE_NO_DEPLOYED" : "Son instance parent n'est pas déployée.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_NOT_DEPLOYED_NO_ACTION" : "L'instance n'est pas déployée. Vous devez déployer et redémarrer son instance parent avant de mettre à jour son état.",
+	
+	"INSTANCES_INCLUDES_DEPLOY_START_IT" : "Déployer et Démarrer l'instance",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_NOT_DEPLOYED" : "L'instance n'est pas déployée. Elle ne peut qu'être déployée.",
+	
+	"INSTANCES_INCLUDES_IP_ADDRESS" : "Adresse IP:",
+	
+	"INSTANCES_INCLUDES_MACHINE_ID" : "Identifiant Machine:",
+	
+	"INSTANCES_INCLUDES_UNDEPLOY_INSTANCE" : "Annuler le déploiement de cette instance",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_ROOT_DEPLOYED_STARTED" : "L'instance est déployée. Elle ne peut être qu'en mode sans déploiement.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_ROOT_DEPLOYING" : "L'instance va être déployée par l'installeur {{installer}} de Roboconf.",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_ROOT_DEPLOYING_2" : "Cela peut prendre quelques temps.",
+	
+	"INSTANCES_INCLUDES_APP_ROOT_PROBLEM" : "L'état de l'instance est indéterminée.",
+	
+	"INSTANCES_INCLUDES_APP_ROOT_PROBLEM_2" : "C'est à dire que le gestionnaire de déploiement de Roboconf ne reçoit pas de signal venant de l'agent associé pendant plus de 2 minutes. Cela peut être expliqué par un problème de réseau du côté de gestionnaire de déploiement, du côté de l'agent, du côté de serveur de messagerie, ou une erreur du serveur de messagerie lui même (assurez vous qu'il fonctionne toujours).",
+	
+	"INSTANCES_INCLUDES_TRY_TO_UNDEPLOY" : "Essayer d'annuler le déploiement de cette instance (Encore)",
+	
+	"INSTANCES_INCLUDES_DESCRIPTION_APP_ROOT_UNDEPLOYING" : "L'instance va être déployée par l'installeur {{installer}} de Roboconf.",
```

Comment:
You want to say "L'instance va être installée" par l'installeur ..."  instead of "L'instance va être déployée par l'installeur .." ?